src/vm/stubmgr.h

```diff
@@ -978,7 +978,7 @@ class StubManagerHelpers
         Thread::VirtualUnwindCallFrame(&context);
         Thread::VirtualUnwindCallFrame(&context);
 
-        return pContext->Rip;
+        return context.Rip;
```

Comment:
Should I change the ARM and ARM64 parts of this code to use the call to VirtualUnwindCallFrame instead of hardcoding some mysterious offsets from registers?

Reply:
I don't know if VirtualUnwindCallFrame works for arm/arm64 properly.  It would be better to use it instead of the hard coded increment but I'm not sure how time you want to spend investigating that.

Reply:
I'm not that familiar with the ARM/ARM64 ABI... is there any reasonable scenario where R11+1 or Fp+1 is not going to give us the right answer?

Reply:
I spoke to @briansull  and he tells me this is actually ok, for the relatively narrow situation where this applies. Since we always generate a fp, lr save in managed prolog's and the function we are unwinding here is always a managed function (without localloc or other odd stuff) that this should always be safe.